integration/docker-compose.xpu.yml

```diff
@@ -116,6 +116,8 @@ services:
       - ./strongswan/strongswan.conf:/etc/strongswan.conf:z
     network_mode: service:xpu-cpu-ssh
     command: './charon'
+    healthcheck:
+      test: swanctl --stats || exit 1
```

Comment:
In general, I think this is good and should work ok.

Reply:
ok